spec/models/manageiq/providers/ibm_power_hmc/infra_manager/event_target_parser_spec.rb

```diff
@@ -0,0 +1,116 @@
+describe ManageIQ::Providers::IbmPowerHmc::InfraManager::EventTargetParser do
+  before :each do
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    @ems                 = FactoryBot.create(:ems_ibm_power_hmc_infra, :zone => zone)
+
+    allow_any_instance_of(EmsEvent).to receive(:handle_event)
+    allow(EmsEvent).to receive(:create_completed_event)
```

Comment:
@vfebvre are these two necessary?  I deleted both of these lines and the specs pass locally just fine.

Reply:
will fix it in PR (#29)